ETL/Provision/test_provision_lambda.py

```diff
@@ -0,0 +1,58 @@
+# pylint: skip-file
+from provision_lambda import group_data, read_database
+from unittest.mock import MagicMock
+import pytest
+
+
+@pytest.fixture
+def fake_readings():
+    return [{"product_id": 1, "url": "http://example.com/product1", "price": 19.99},
+            {"product_id": 2, "url": "http://example.com/product2", "price": 24.99},
+            {"product_id": 3, "url": "http://example.com/product3", "price": 12.34},
+            {"product_id": 4, "url": "http://example.com/product4", "price": 228.99}]
+
+
+def test_group_data_batch_of_2(fake_readings):
+    readings = fake_readings
```

Comment:
Add docstrings to each function explaining the function

Reply:
Sure thing😎